recipes/pg_master.rb

```diff
@@ -27,35 +27,25 @@
 node.set['postgresql']['config']['checkpoint_segments'] = 8
 node.set['postgresql']['config']['wal_keep_segments'] = 8
 
-# set SSL usage based on OS support (debian yes, Redhat no)
+# since debian OS families can do ssl by default enable it, otherwise non-ssl connections
 case node['platform_family']
```

Comment:
Since you are using chef-sugar in this recipe already, you might be interested in these: `debian?`, `rhel?` from https://github.com/sethvargo/chef-sugar/blob/master/lib/chef/sugar/platform_family.rb.